packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -29,9 +31,11 @@ const SubscriptionUtils = (module.exports = {
   determineClientVisibleSubscriptionCapabilities: async function(
     stripeHelper,
     uid,
-    client_id,
+    clientIdRaw,
     email
   ) {
+    const clientId =
+      clientIdRaw === null ? null : hex(clientIdRaw).toLowerCase();
```

Comment:
Maybe add comment here to mention that if the client id is already a string, the `hex` call will return it as it is.